README.md

```diff
@@ -154,6 +203,57 @@ We enthusiastically welcome contributions and feedback. You should clone the rep
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+### Clone
```

Comment:
What Is this section? Can it go above the community help and support section also can there be some more explanation?

Reply:
Maybe we stick this in the Contributing section off with other stuff like 'community guidelines', legalese, and whatever else may come later